latest.yaml

```
@@ -0,0 +1,5 @@
+# Define the values used for the "latest" tag
+LATEST_VERSIONS:
+  CUDA_VER: "12.0.1"
+  PYTHON_VER: "3.10"
+  LINUX_VER: "ubuntu22.04"
```

```suggestion
conda:
  CUDA_VER: "12.0.1"
  PYTHON_VER: "3.10"
  LINUX_VER: "ubuntu22.04"
wheels:
  CUDA_VER: "12.0.1"
  PYTHON_VER: "3.10"
  LINUX_VER: "ubuntu20.04"
```

Let's use this file as the source of truth for the `latest` tags for both `wheel` and `conda` images.